plugins/exodus/exodus_

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+"""
+=head1 NAME
+
+exodus_ - Exodus wildcard-plugin to monitor an L<Exodus|https://github.com/Exodus-Privacy/exodus/>
+instance.
+
+This wildcard plugin provides at the moment only the suffixes C<applications>, C<reports>
```

Comment:
Maybe remove `only` here? Or state, which other parts you consider to be missing.